### ISSUE TITLE
"TypeError: undefined is not a function" when creating new project

### ISSUE THREAD

**Issue by mbabauer** [mantrajs/mantra-cli#21]
I installed the latest mantra-cli via npm, and when I try to create a new project I get the following:

```
mantra create logApp_web
  create  logApp_web/
  invoke  init

/usr/local/lib/node_modules/mantra-cli/dist/utils.js:155
  (0, _child_process.execSync)('bash ' + pathToScript + ' ' + args.join(' '),
                              ^
TypeError: undefined is not a function
    at runScriptFile (/usr/local/lib/node_modules/mantra-cli/dist/utils.js:155:31)
    at create (/usr/local/lib/node_modules/mantra-cli/dist/commands/create.js:28:44)
    at Command.<anonymous> (/usr/local/lib/node_modules/mantra-cli/dist/cli.js:18:24)
    at Command.listener (/usr/local/lib/node_modules/mantra-cli/node_modules/commander/index.js:301:8)
    at Command.emit (events.js:98:17)
    at Command.parseArgs (/usr/local/lib/node_modules/mantra-cli/node_modules/commander/index.js:615:12)
    at Command.parse (/usr/local/lib/node_modules/mantra-cli/node_modules/commander/index.js:458:21)
    at Object.<anonymous> (/usr/local/lib/node_modules/mantra-cli/dist/cli.js:38:21)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
```

Specs are as follows:
- OS X vers. 10.11.2
- npm vers. 1.4.21
- Node vers. 0.10.30
- Mantra CLI vers. 0.3.0 (according to npm view)

I should also say that I installed Mantra CLI as root, as it gave errors when I tried to install it as my normal user.  Also, I am using zsh, which I only mention because the error references bash, although bash still works.

**Comment by sungwoncho** [comment]
In node v0.10.x, `child_process.execSync` is not available. That is why we get `undefined is not a function`. It was added in v0.12.0 ([see here](https://nodejs.org/en/blog/release/v0.12.0/)). I will welcome a PR or a suggestion for backward compatibility.

For the installation issue, I think you need to check that the normal user has permission to write at the directory that npm is installing the module.

**Comment by sungwoncho** [comment]
Fixed in 0.3.1. See https://github.com/mantrajs/mantra-cli/pull/18
